hack/build-local-images.py

```diff
@@ -127,7 +127,18 @@
             "openshift": "/usr/bin/openshift"
         },
         "files": {}
-    }
+    },
+    "service-catalog": {
+        "directory": "service-catalog",
+        "vendor_dir": "cmd/service-catalog/go/src/\
```

Comment:
formatting seems silly

Reply:
Maybe so, was just trying to not add additional lines greater than 80 characters (technically 79 is the standard). There are different ways for handling multiline strings - what's your preference?

Reply:
Don't worry about it. Python formatting is an awful morass of opinions ... but treat it like Go? I don't feel super strongly